jupyterlab/labbox_ephys_widgets_jp/src/extensions/unitstable/Units/UnitsTable.tsx

```diff
@@ -109,12 +102,22 @@ const interpretSortFields = (fields: string[]): sortFieldEntry[] => {
 }
 
 const UnitsTable: FunctionComponent<Props> = (props) => {
-    const { sortingUnitMetrics, units, metrics, selectedUnitIds, sorting, onSelectedUnitIdsChanged } = props
+    const { sortingUnitMetrics, units, metrics, selection, selectionDispatch, sorting } = props
     const sortingUnitMetricsList = sortByPriority(Object.values(sortingUnitMetrics)).filter(p => (!p.disabled))
     const [sortFieldOrder, setSortFieldOrder] = useState<string[]>([])
     units.sort((a, b) => a - b) // first sort by actual unit number
     // Now sort the list iteratively by each of the sorters in the sortFieldOrder state.
 
+    const toggleSelectedUnitId = useCallback(
```

Comment:
At some point, toggle can be handled in the sortingSelectionReducer for better performance. But for now this will be fine.